pkg/api/v1alpha1/cloudstackmachineconfig_webhook.go

```diff
@@ -53,6 +53,11 @@ func (r *CloudStackMachineConfig) ValidateUpdate(old runtime.Object) error {
 		return nil
 	}
 
+	if oldCloudStackMachineConfig.IsManagement() {
```

Comment:
This check alone is actually redundant. I was referring to having a list of fields that we don't want immutable, but we can have it as a follow up before we remove the feature flag if you can log it as an issue.

https://github.com/aws/eks-anywhere/blob/main/pkg/api/v1alpha1/vspheremachineconfig_webhook.go#L58

Reply:
https://github.com/aws/eks-anywhere/issues/1669